mod/wikirate_source/spec/set/type/source/source_preview_spec.rb

```diff
@@ -49,7 +49,10 @@
         it "shows pdf" do
           file_card = @pdf_source.fetch :trait=>:file
           expect(@result).to have_tag("div", :with=>{:id=>"pdf-preview"}) do
-            with_tag "iframe", :with=>{:id=>"source-preview-iframe", :src=>"pdfjs/viewer.html?file=#{file_card.attachment.url}"}
+            with_tag "iframe", with: {
+                                id: "source-preview-iframe", 
+                                src: "files/viewer.html?file=#{file_card.attachment.url}"
+                              } 
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.<br>Trailing whitespace detected.